### PR TITLE
DOC: Add formula renderings to integrate.nquad.

### PR DESCRIPTION
**Pull request by derbuihan**
#### What does this implement/fix?

Improved DOC for nquad.
Added rendering of formulas and made it easier to read.

### CODE REVIEW

**Review by melissawm — COMMENTED** [review]
Hi @derbuihan , I left a few comments to fix the tests that are failing.

**Comment by derbuihan** [comment]
@melissawm 
I applied all the suggestions.

**Comment by derbuihan** [comment]
I do not know how to pass the test.

**Comment by ilayn** [comment]
You need to turn off the backslash escaping which is by default enabled on `"""` - `"""` string blocks. But TeX syntax involves backslashes so on line 730 instead of `"""` change it to an r string via `r"""`

File: scipy/integrate/_quadpack_py.py [hunk 1]
```diff
@@ -764,16 +764,16 @@ def nquad(func, ranges, args=None, opts=None, full_output=False):
...
     opts : iterable object or dict, optional
         Options to be passed to `quad`. May be empty, a dict, or
         a sequence of dicts or functions that return a dict. If empty, the
         default options from scipy.integrate.quad are used. If a dict, the same
         options are used for all levels of integraion. If a sequence, then each
         element of the sequence corresponds to a particular integration. e.g.,
-        opts[0] corresponds to integration over x0, and so on. If a callable,
-        the signature must be the same as for ``ranges``. The available
+        ``opts[0]`` corresponds to integration over ``x0``, and so on. If a
+        callable, the signature must be the same as for ``ranges``. The available
```
Comment (ilayn) [inline scipy/integrate/_quadpack_py.py:776]: This line is a bit longer than 80 characters tripping linter.

Comment (derbuihan) [reply scipy/integrate/_quadpack_py.py]: thank you.

**Comment by derbuihan** [comment]
@melissawm, @ilayn 
Passed the test. Thanks for your review.

**Review by melissawm — APPROVED** [review]
This looks good to me, thanks @derbuihan !

**Comment by ilayn** [comment]
Let's get this in then. Thanks @derbuihan @melissawm !

**Pull request merged**